README.md

```diff
@@ -9,11 +9,31 @@ This project aims an update of Remoting and Jenkins in order to add support of a
 
 More information about this project can be found at: https://jenkins.io/projects/gsoc/2018/remoting-over-message-bus/
 
-## Requirement
-docker, docker-compose installed
+## Release Notes
+
+See the [Changelog](CHANGELOG.md).
+
+## How to use the plugin in alpha version
+
+Alpha version of the plugin is now released under [Experimental Update Center](https://jenkins.io/doc/developer/publishing/releasing-experimental-updates/#configuring-jenkins-to-use-experimental-update-center).
+
+Requirements to use the plugin under alpha released:
+
+1. You must have a Kafka cluster running (with host and port). If not, you can up Kafka and Zookeeper services using `docker-compose.yml` in this repository.
+
+2. You must have a custom agent JAR to use this plugin. The JAR can be built from [agent](https://github.com/jenkinsci/remoting-kafka-plugin/tree/master/agent) module in this repository.
```

Comment:
I would replace it by "download from artifactory". Ideally the plugin could offer the executable download from the Jenkins page